apps/lxp/modules/profile/main/edit-profile/section.tsx

```diff
@@ -26,8 +31,8 @@ export const EditProfileSection = () => {
   ];
 
   const genders = [
-    { id: 1, value: 'L', label: 'Laki-Laki' },
-    { id: 2, value: 'P', label: 'Perempuan' },
+    { id: 1, value: 'LAKI-LAKI', label: 'Laki-Laki' },
```

Comment:
Kenapa di capslock dan ga disingkat aja ya?

Reply:
Dari BE minta gitu LAKI-LAKI sama PEREMPUAN

Reply:
biar lngsng value nya sama kaya yg diminta be